ISISComputingGroup/EPICS-motionSetPoints
# Ticket 4573: SANS2D, Named aperture and guide positions

See https://github.com/ISISComputingGroup/IBEX/issues/4573.

In doing this support was also removed for non-axis motors, see https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Decision-Log

Also fixes (close or comment on these as appropriate):

- [x] Part of https://github.com/ISISComputingGroup/IBEX/issues/2895
- [x] https://github.com/ISISComputingGroup/IBEX/issues/3989 (referenced PVs no longer exist)
- [x] https://github.com/ISISComputingGroup/IBEX/issues/4689
- [x] https://github.com/ISISComputingGroup/IBEX/issues/3967

To test:
* Run the new unit tests by calling `make runtests` in the top directory (you will need https://github.com/ISISComputingGroup/IBEX/issues/5513)
* Run the new IOC system tests (https://github.com/ISISComputingGroup/EPICS-IOC_Test_Framework/pull/360)
* On the SANS2D configuration branch confirm that the aperture and waveguide named positions work
